### PR TITLE
Fixed #24817 - Prevented loss of null info in MySQL field renaming

### PR DESCRIPTION
**Pull request by coldmind**
Previous PR - https://github.com/django/django/pull/4714
Opening against master.

https://code.djangoproject.com/ticket/24817

### CODE REVIEW

File: django/db/backends/mysql/schema.py [hunk 1]
```diff
@@ -80,10 +80,20 @@ def _delete_composed_index(self, model, fields, *args):
                 )
         return super(DatabaseSchemaEditor, self)._delete_composed_index(model, fields, *args)
 
-    def _alter_column_type_sql(self, table, old_field, new_field, new_type):
-        # Keep null property of old field, if it has changed, it will be handled separately
-        if old_field.null:
+    def _set_field_new_type_null_status(self, field, new_type):
+        """
+        Keep null property of old field, if it has changed, it will be handled separately
```
Comment (timgraham) [inline django/db/backends/mysql/schema.py]: Keep the null property of the old field. If it has changed, .... separately.
(also please wrap at 79 chars)

**Comment by timgraham** [comment]
Could you please add 1.8.3 release notes?

**Comment by coldmind** [comment]
@timgraham sure, just added it.

**Comment by timgraham** [comment]
merged in 80ad5472ce4b6ba6e94227422d0727371e97cdf0, thanks!

**Pull request closed without merging**